cloudfront/wellcomelibrary.org/edge-lambda/testRedirects.ts

```diff
@@ -179,11 +178,23 @@ const blogTestSet = {
 const apexTestSet = {
   displayName: 'Apex (Content management) pages',
   fileLocation: staticRedirectFileLocation,
-  fileHostPrefix: staticRedirectsHost,
-  headers: staticRedirectHeaders,
+  fileHostPrefix: 'blog.wellcomelibrary.org',
+  headers: ['sourceUrl', 'targetUrl'],
   envs: {
-    stage: 'https://stage.wellcomelibrary.org',
-    prod: 'https://wellcomelibrary.org',
+    stage: 'https://blog.stage.wellcomelibrary.org/',
```

Comment:
Why has this become `blog.`?

Reply:
No idea what I was trying to do here, reverted.